components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.device-organization/device-organization.hbs

```diff
@@ -0,0 +1,66 @@
+{{!
+  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please change the license header's year.

Reply:
Fixed 9d4b1663c67e34add2e5e6ce2c83150768f4fee9